### ISSUE TITLE
Missing C>T variants in VarDictJava

### ISSUE THREAD

**Issue by tinyheero** [AstraZeneca-NGS/VarDictJava#194]
Hi,

I am struggling to understand why VarDict is not calling a set of mutations in a dataset. For instance, here is an IGV screenshot of one particular mutation:

<img width="1438" alt="igv_screenshot" src="https://user-images.githubusercontent.com/3268173/53494339-6de87600-3a95-11e9-80fe-5c7a6267e923.png">

This mutation has a variant allele fraction (VAF) of about 0.19, which is not that low. I assumed VarDict would report this mutation in the raw data. But when running VarDict on this tumour-normal pair:

```bash
VarDict \
        -b "tumour.bam|normal.bam" \
        -f 0.001 \
        -G ucsc.hg19-GATKBundle.fa \
        -c 1 -S 2 -E 3 \
        -N tumour_normal \
        -th 1 \
        regions.bed \
    > tumour_normal_vardict.var
```

This mutation is not reported in the raw results of VarDict. In fact, there are several mutations that I am interested in that are also not predicted by VarDict:

* chr2:37406700_C>T
* chr3:169497172_C>T
* chr7:98506465_C>T
* chr7:100730873_C>T
* chr13:32351545_C>T
* chr17:12823136_C>T
* chrX:138672083_C>T

All of these mutations are C>T and have a VAF between 0.1 to 0.2. I originally ran this using VarDictJava version 1.5.1-g514215a. But I decided to give it a try using the latest version of VarDictJava 1.5.8 and suddenly all of the missing mutations are now being reported. Based on the release notes, it's not clear to me what has changed from 1.5.1 to 1.5.8 that has resolved this. Does anyone have any ideas?

I've added the following file for reproducibility purposes to https://drive.google.com/drive/folders/1kKsUny5wf6WVBrCfy4Flhr399k2ng4gY:

* `tumour.{bam,bai}`: The tumour BAM and BAI file that has been subsetted to regions that contain the above mutations.
* `normal.{bam,bai}`: The normal BAM and BAI file that has been subsetted to regions that contain the above mutations.
* `regions.bed`: Regions that contain the mutation.

**Comment by PolinaBevad** [comment]
Fong Chun Chan, hello,

Thank you for preparing files for testing!
There were some changes in version 1.4.8 of VarDictJava about skipping overlapping reads due to this discussion: https://github.com/AstraZeneca-NGS/VarDictJava/issues/60. But as was found later (example in issue https://github.com/AstraZeneca-NGS/VarDictJava/issues/102) some of true variants were skipped and so it is was fixed in version 1.5.3 (now skipping is working with options `-u` and `-UN` for R1/R2 and forward/reverse cases). That's the reason for this behavior.

**Comment by tinyheero** [comment]
Thanks @PolinaBevad for the explanation. Much appreciated.
